### PR TITLE
B2B Styling

### PR DESCRIPTION
**Pull request by paddyo**
## Description
Styling and layout for the B2B prospect trial form

## Prerequisites
https://github.com/Financial-Times/next-signup/pull/765
https://github.com/Financial-Times/n-barriers/pull/28

### CODE REVIEW

File: views/partials/barriers/corporate-prospect-trial.html [hunk 1]
```diff
@@ -1,24 +1,79 @@
...
+					<span class="barrier-grid__pretext">Already a subscriber?</span>
+					<a data-trackable="sign-in" href="/login">Sign In</a>
+				</div>
+			</div>
+
+			<div data-o-grid-colspan="12 M6">
+				<div class="corporate-prospect-trial__description">
+					<img src="{{#resize 230}}https://next-geebee.ft.com/assets/tour/pack-shot-2.png{{/resize}}" width="230" height="auto" alt="" class="su-header__image">
+					<h1 class="corporate-prospect-trial__title">You've been invited</h1>
+					<p>This month, employees of your company can enjoy unlimited access to the FT.com, including:</p>
```
Comment (phamann) [inline views/partials/barriers/corporate-prospect-trial.html]: This month? Maybe we should avoid mentioning any time context?

File: views/partials/barriers/corporate-prospect-trial.html [hunk 2]
```diff
@@ -1,24 +1,79 @@
...
+						}
+
+						return () => {
+							if(!running) {
+								running = true;
+								window.requestAnimationFrame(op);
+							}
+						}
+
+					})();
```
Comment (phamann) [inline views/partials/barriers/corporate-prospect-trial.html]: Lot of use of ES6 in this function. As this is being ran inline within the document we'll have to stick to ES3 😬

Comment (paddyo) [inline views/partials/barriers/corporate-prospect-trial.html]: Without complication/polyfilling the iframe code simply won't work, will move to a file

Comment (paddyo) [inline views/partials/barriers/corporate-prospect-trial.html]: I forgot we don't need to support IE8 anymore, so I've devolved it.

**Pull request merged**